src/umockcallrecorder.c

```diff
@@ -829,3 +829,23 @@ int umockcallrecorder_can_call_fail(UMOCKCALLRECORDER_HANDLE umock_call_recorder
 
     return result;
 }
+
+char* umockcallrecorder_get_call_description(UMOCKCALLRECORDER_HANDLE umock_call_recorder, size_t index)
```

Comment:
>description [](http://example.com/codeflow?start=33&length=11)

Poetry: This smells like umockcallrecorder_get_expected_call_string.

That's because it only gets expected call strings (as opposed to actual call strings that are also stored by the recorder).
Re string vs description, I think string fits better because stringifying the call might not provide a lot of information besides the function name and some arguments. Description usually makes me think more of an essay like thing that details what the function does for example.

Reply:
Your poetry sounds true in the ears of this humble listener.
The name shall be changed to rhyme better.

Reply:
Done.